README.md

```diff
@@ -94,3 +95,12 @@ sudo apt update
 sudo apt install chromium-browser
 ```
 For more details and instructions on possibly required driver updates see [here](https://www.linuxuprising.com/2018/08/how-to-enable-hardware-accelerated.html).
+
+## Keyboard backlight
```

Comment:
I just installed a plain vanilla Ubuntu 19.10 and the firmware update suggested by the Software/Updates tool coming with Ubuntu. Then the keyboard backlight seems to work out of the box.

Reply:
How strange ... just installed either completely from scratch; I did a minimal install (no games, office, ...)

Which is (name/version) the firmware package?

Reply:
I also used the minimal version. The firmware package name I don't know, because I clicked the "yes, whatever" button too quickly. But it updated the BIOS upon the next restart, so don't install it on an empty battery.

The way I found the update was by starting the Ubuntu Software manager. It has three buttons at the top "All", "Installed" and "Updates". The latter section showed a firmware update.

Reply:
You are right: removed the parameter from /etc/default/grub and the keyboard lighting still works, I guess it's thanks to BIOS 1.3.3

Reply:
BTW Why you removed the firs two lines in the script? I still need them on my pc.

Reply:
Yes, you're right, that came from another PR. I have readded them, but without the `export`. This way the script should work without breaking `xrandr` for other uses.

Reply:
So it can be merged now I guess